docs/src/running-validator/validator-troubleshoot.md

```diff
@@ -15,3 +15,31 @@ testnet participants, [https://discord.gg/pquxPsq](https://discord.gg/pquxPsq).
 - [Core software repo](https://github.com/solana-labs/solana)
 
 Can't find what you're looking for? Send an email to ryan@solana.com or reach out to @rshea\#2622 on Discord.
+
+## Blockstore
+
+The validator blockstore rocksdb database can be inspected using the `ldb` tool.
+`ldb` is part of the `rocksdb` code base and is also available in the `rocksdb-tools`
+package.
+
+[RocksDB Administration and Data Access Tool](https://github.com/facebook/rocksdb/wiki/Administration-and-Data-Access-Tool)
+
```

Comment:
For completeness, we could add a section about upgrade (which we handle automatically). Doing so might also help illuminate how an upgrade/downgrade sequence puts blockstore into bad state for old software.
```
## Upgrade

If a new software version introduces a new column family to the blockstore,
that new (empty) column will automatically be created. This is the same logic
that allows a validator to start without the blockstore directory.
```